<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Function Name:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <layout class="QGridLayout" name="fit_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
    </layout>
   </item>
   <item row="5" column="5">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>134</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QComboBox" name="fit_function_selector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QPushButton" name="output_results_table_btn">
     <property name="text">
      <string>Output Results</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="results_name_editor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Log Values</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Table Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Fit Results</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="6">
    <layout class="QGridLayout" name="log_value_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
    </layout>
   </item>
  </layout>
  <zorder>fit_function_selector</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>output_results_table_btn</zorder>
  <zorder>horizontalSpacer_3</zorder>
  <zorder>results_name_editor</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
